server/chat-commands/info.js

```diff
@@ -2339,6 +2339,18 @@ const commands = {
 	},
 	pickrandomhelp: [`/pick [option], [option], ... - Randomly selects an item from a list containing 2 or more elements.`],
 
+	'!shuffle': true,
+	shuffle(target, room, user) {
+		if (!target) return false;
+		if (!target.includes(',')) return this.parse('/help shuffle');
+		if (!this.runBroadcast(true)) return false;
+		let args = target.split(',');
```

Comment:
```suggestion
		const args = target.split(',');
```

it's never reassigned